administrator/language/de-DE/de-DE.com_fields.ini

```diff
@@ -9,7 +9,9 @@ COM_FIELDS_BATCH_GROUP_LABEL="Zum Verschieben oder Kopieren der Auswahl eine Gru
 COM_FIELDS_BATCH_GROUP_OPTION_NONE="- Keine Gruppe -"
 COM_FIELDS_ERROR_UNIQUE_NAME="Ein anderes Feld hat denselben Namen (ggf. auch im Papierkorb)."
 COM_FIELDS_FIELD_CLASS_DESC="Das CSS-Klassenattribut für das Feld in der Eingabemaske. Bei mehreren Klassen mit Leerzeichen trennen."
```

Comment:
Das hier ggf. als Beispiel nehmen? & Wenn wir Eingabe Klasse in Feldklasse ändern sollten wir das hier auch machen.

Reply:
Den Gedanken hatte ich auch grad 👍